src/graphnet/models/graphs/utils.py

```diff
@@ -149,6 +151,11 @@ def cluster_summarize_with_percentiles(
     Returns:
         Percentile-summarized array
     """
+    print(
+        "This function is deprecated and will be removed,",
+        "use the class cluster_and_pad with add_percentile_summary",
+        "instead for the same functionality",
+    )
```

Comment:
While it's good to warn the user that this function is deprecated, this will print for every call (lots of printing). I am not sure what a good solution is here.

Reply:
Great catch! I missed that.

Reply:
I have removed the warning but kept the TODO and function for now. If we all agree to remove the function then I am not against it but just prefer to air on the side of caution.